pyfar/classes/audio.py

```diff
@@ -669,16 +656,18 @@ def domain(self, new_domain):
             # Only process if we change domain
```

Comment:
Lets simplify the line above to `if self._domain != new_domain`

Reply:
changed it :)